### PR TITLE
 fix journal internet access links

### LINKED ISSUE #393: No Internet Access links appearing for journal titles in the public NEOS Catalogue
**Describe the bug**
From a user report June 19, 2020
>I have looked at several journal records in the NEOS catalogue to find that the Internet Access link is not appearing. Therefore, there is no way for our users to access the resources on this platform. I have found that the bulk loaded records for ebooks and streaming videos have links that work, but individual journal records are not showing links for any of the institutions listed. 

>I have not looked into individually purchased streaming videos or ebooks, so I can't say if there is an issue with those types of records as well. 

>This may not be a big deal at the moment, but it will become important when students attempt to gain access to these resources for their courses in the Fall term. 

>For instance, these subscribed journals have no links:
> - Journal of Advanced Nursing
> - Zero-to-three
> - Journal of transcultural nursing
> - Research in gerontological nursing
> - Nursing science quarterly
> - and many, many more. 

>We have had issues with links not appearing for Free Access resources in the past, but we have not had issues with our subscribed journal title links not appearing. Any idea what is going on with the catalogue?

**To Reproduce**
Visit https://catalogue.neoslibraries.ca/catalog/6821744.  
image

Would expect that there be links to the resources like this older version of the application:
image

**Additional context**
Ticket#202006191022
New monthly index June 16, 2020.
NEOSDiscovery release 1.0.60 June 8
June 20 report that log growing rapidly.

### PR DESCRIPTION
**Pull request by pgwillia**
It appears there is some variation in how the link is presented.  It seems that the links that weren't showing up had a different construction of the link key coming from Symphony

image

closes #393

### CODE REVIEW

File: spec/features/item_holding_table_spec.rb [hunk 1]
```diff
@@ -19,15 +20,22 @@
       expect(first('#holdings table td')).to have_text('status: internet')
       expect(first('#holdings table td')).to have_css("a[href='https://www.burmanu.ca/library']")
 
-      visit '/catalog/3951085'
-      page.assert_selector('#holdings table td', count: 3)
-      expect(first('#holdings table td')).to have_text('Grande Prairie Regional College - Internet')
-      expect(first('#holdings table td')).to have_link('Click here for Internet Access', href: 'http://www.cbd.int/doc/bioday/2007/ibd-2007-booklet-01-en.pdf')
+      visit '/catalog/1007482'
```
Comment (pgwillia) [inline spec/features/item_holding_table_spec.rb:23]: Replacing one instance with an example from the discovery test set

File: spec/features/item_holding_table_spec.rb [hunk 2]
```diff
@@ -19,15 +20,22 @@
...
+      page.assert_selector('#holdings table td', count: 2)
+      expect(first('#holdings table td')).to have_text('University of Alberta Internet')
+      expect(first('#holdings table td')).to have_link('Click here for Internet Access', href: 'http://www.peabody.yale.edu/scipubs/bulletins_postillas/ypmB04_1935.pdf')
       expect(first('#holdings table td')).to have_text('call number: Internet Access')
       expect(first('#holdings table td')).to have_text('status: internet')
-      expect(first('#holdings table td')).to have_css("a[href='https://www.gprc.ab.ca/library']")
+      expect(first('#holdings table td')).to have_css("a[href='https://www.library.ualberta.ca/services/off-campus-access']")
+
+      visit '/catalog/2714245'
```
Comment (pgwillia) [inline spec/features/item_holding_table_spec.rb:31]: This is an example of the type of response that was identified by the ticket

File: spec/features/item_holding_table_spec.rb [hunk 3]
```diff
@@ -19,15 +20,22 @@
...
+      page.assert_selector('#holdings table td', count: 1)
+      expect(first('#holdings table td')).to have_text('Alberta Government Library - Internet')
+      expect(first('#holdings table td')).to have_link('Click here for Internet Access', href: 'https://ezproxy.aee.talonline.ca/login?url=http://purl.access.gpo.gov/GPO/LPS31752')
+      expect(first('#holdings table td')).to have_text('call number: Internet Access')
+      expect(first('#holdings table td')).to have_text('status: internet')
+      expect(first('#holdings table td')).to have_css("a[href='https://www.servicealberta.ca/alberta-government-library.cfm']")
 
-      page
```
Comment (ConnorSheremeta) [inline spec/features/item_holding_table_spec.rb:30]: Any idea why this was here?

Comment (pgwillia) [reply spec/features/item_holding_table_spec.rb:30]: 🤷‍♀️

**Pull request merged**